src/Makefile.m32

```diff
@@ -238,13 +238,20 @@ endif
 ifeq ($(findstring -ngtcp2,$(CFG)),-ngtcp2)
 NGTCP2 = 1
 endif
+ifeq ($(findstring -unicode,$(CFG)),-unicode)
+UNICODE = 1
+endif
 
 INCLUDES = -I. -I../include -I../lib
 ifdef SSL
   ifdef WINSSL
     CFLAGS += -DCURL_WITH_MULTI_SSL
   endif
 endif
+ifdef UNICODE
+  CFLAGS += -DUNICODE -D_UNICODE
+  LDFLAGS += -municode
```

Comment:
If this is needed why is it only used in src not lib?

Reply:
`-municode` tells the linker to use init code that calls `wmain()`, so it's only needed for `curl.exe`. More details on this: <https://stackoverflow.com/a/11706847/1732433>